Zeeguu Reader/Zeeguu Reader/FeedOverviewTableViewController.swift

```diff
@@ -44,6 +44,8 @@ class FeedOverviewTableViewController: ZGTableViewController, AddFeedTableViewCo
 	override func viewDidLoad() {
 		super.viewDidLoad()
 		
+		NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(FeedOverviewTableViewController.userDidLogin(_:)), name: UserLoggedInNotification, object: nil)
```

Comment:
Try this: https://github.com/mdo/github-wide ;)
![schermafbeelding 2016-06-10 om 12 46 03](https://cloud.githubusercontent.com/assets/4348209/15962126/5094d1ce-2f09-11e6-9702-8810de252aa8.png)